tasks/service.yml

```diff
@@ -13,3 +13,15 @@
     port: "{{ grafana_port }}"
     state: "{{ grafana_service_state }}"
     timeout: 30
+
+- name: GRAFANA | Wait for health respond 200
+  uri:
+    url: "http://{{ ansible_nodename }}:{{ grafana_port }}/api/health"
+    status_code: "200"
+    method: GET
+    headers:
+      Accept: application/json
+  register: result
+  until: result.status == 200
+  retries: 60
+  delay: 1
```

Comment:
maybe I would increase this numbers or make it configurable via vars, because we don't know how much time will be needed in a real world installation with a lot of dashboards...

Reply:
Added new timeout variable.